src/app/app.js

```diff
@@ -1,4 +1,6 @@
 // Special Debug Console Calls!
+var localStorage = window.localStorage || {};
```

Comment:
nope, but it does it on nwjs 0.18 too, so it's either an old bug, or something else